### ISSUE TITLE
saba docker 1.10 installation

### ISSUE THREAD

**Issue by hxrts** [mskcc/HPC-UserGroup#10]
Docker's former security escalation issue has now been resolved with namespaced environments in the 1.10 release. More information can be found (here)[http://www.infoworld.com/article/3030558/application-virtualization/docker-goes-rootless-and-thats-a-good-thing.html]. This is something that would make our work much more maintainable and replicable while simultaneously reducing the overhead of system admins. I'd strongly encourage us to consider implementing (this release)[https://blog.docker.com/2016/02/docker-1-10/] on saba.

**Comment by ckandoth** [comment]
![everything is awesome](http://45.media.tumblr.com/03e1512af92856988dc7b44a2a263796/tumblr_n0yewddtMd1rf8id2o1_400.gif)

**Comment by lordzappo** [comment]
docker was a long term milestone for the portability of CMO Haystack
pipelines as well, so this would benefit us w.r.t. the goals of ultimate
portability of pipelines to the NYGC and for reproducibility in all future
paper algorithms/pipelines we ship.


CCH

**Comment by ckandoth** [comment]
I was super happy until I remembered that all our compute nodes use CentOS 6 with v2.6 kernels. Officially, docker supports v3.10 or newer kernels, which is in CentOS 7. Same story with RedHat - supporting docker on RHEL7. Still worth isolating a box to test it and see what breaks.

**Comment by jchodera** [comment]
+1 for this solution on Hal as well. I understand that @juanperin is working on an upgrade plath to newer CentOS versions to support this.

**Comment by juanperin** [comment]
That’s right, we’re working on it.  There is an ongoing effort to try RHEL7 and ensure we know what else is different.  Hopefully we can upgrade very soon. 

Juan

**Comment by ckandoth** [comment]
@juanperin Glad to hear. Let me know how I can help.
